package.json

```diff
@@ -17,42 +17,40 @@
   "license": "MIT",
   "dependencies": {
     "@leizm/web": "^1.7.1",
-    "autocannon": "^2.0.0",
+    "autocannon": "^2.4.1",
     "autocannon-compare": "^0.3.0",
-    "chalk": "^2.3.2",
-    "cli-table": "^0.3.1",
-    "commander": "^2.14.1",
+    "chalk": "^2.4.1",
+    "commander": "^2.15.1",
     "connect": "^3.6.6",
     "cors": "^2.8.4",
     "dns-prefetch-control": "^0.1.0",
-    "express": "^4.16.2",
-    "fastify": "^1.5.0",
+    "express": "^4.16.3",
+    "fastify": "^1.6.0",
     "frameguard": "^3.0.0",
-    "hapi": "^17.5.0",
+    "hapi": "^17.5.1",
     "hide-powered-by": "^1.0.0",
     "hsts": "^2.1.0",
     "ienoopen": "^1.0.0",
-    "inquirer": "^5.2.0",
-    "koa": "^2.5.0",
+    "inquirer": "^3.3.0",
```

Comment:
@mcollina -Is there a reason why some of the packages have been downgraded here? -Also, any particular reason as to why `rayo` was removed? -Might as well remove it from the results table if you don't see it fit.

Reply:
I probably messed up with the rebase, let me check.